util/logging.go

```diff
@@ -97,13 +97,13 @@ func LoggingMiddleware(logger *zap.Logger) mux.MiddlewareFunc {
 	logger = logger.Named("http").WithOptions(zap.WithCaller(false))
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			// Do not log requests to the health endpoint
-			if r.RequestURI == "/health" {
+			switch r.RequestURI {
+			case "/health":
+				// Do not log requests to these endpoints
 				next.ServeHTTP(w, r)
-				return
+			default:
+				logRequest(logger, next, w, r)
 			}
-
-			logRequest(logger, next, w, r)
```

Comment:
Frankly speaking, I would log all requests. At least we know that some script/bot/logic/ hits the healthcheck. WDYT?

Reply:
For context, I just changed the code here because I wanted to filter outlogs from /metrics too. As part of one of the comments now /metrics is under another host:port (http.ListenAndServe()), so at the end it was not needed.

Not a strong opinion here, I see that this filter in case of /health endpoint could be advantageous to inspect logs later, since it could flood elastic with too many GET requests to /health (they could be filtered in a KQL query too). However, this is hiding from us the actual requests (in logs) to all the endpoints.

I could remove it and log all requests, and check whether or not it causes a problem when we try to diagnose, check logs... Ans if that is the case, we could remove it.
WDYT ?

Reply:
> I got tempted by this when migrating to the ECS logger, but it flooded the logs quite a bit. Let's leave it for a separate PR if we do it.

Ok, I'll keep the same logic and let's try that in a separate PR.